.github/workflows/gateway-conformance.yml

```diff
@@ -74,7 +74,7 @@ jobs:
           kuboNodeMultiaddr=$(ipfs --api=/ip4/127.0.0.1/tcp/5001 swarm addrs local --id | head -n 1)
 
           # run gw
-          ./rainbow --routing=http://127.0.0.1:8080 --peering=$kuboNodeMultiaddr &
+          ./rainbow --routing-v1-endpoints=http://127.0.0.1:8080 --dht-routing=false --peering=$kuboNodeMultiaddr &
```

Comment:
🙋‍♂️ Why were we peering it with the Kubo node if `--routing` automatically disabled any DHT routing? What benefit would that bring?

Reply:
Ah, I think I answered myself: Kubo has all the fixtures here and Rainbow still needs to talk to Kubo to get the actual data. This just ensures that we're connected to the Kubo node.